en/appendices/3-5-migration-guide.rst

```diff
@@ -135,6 +135,15 @@ behavior that may affect your application:
 New Features
 ============
 
+Scoped Middleware
+-----------------
+
+Middleware can now be conditionally applied to only routes in specific URL
```

Comment:
"to only routes in specific URL scopes."
The "only" seems to be misplaced. Not an English native speaker, though.

"to routes in specific URL scopes, only."
May be? Sounds more correct to me, at least.